### DISCUSSION TITLE
Possibility to sync, but only upload changes as a final sync

### DISCUSSION THREAD

**Discussion by slagtejn** [Mimetis/Dotmim.Sync#704] [Ideas]
We have a scenario if you close the application it should perform a final sync.  This is possible by just syncing the scopes, but would it be possible to only perform an upload sync session?  When the application closes we only care about changes going to the server.

**Comment by Mimetis** [comment]
I don't know honestly
Doing that kind of action will break the sync between server and client, and you client may become "un-synced"

I guess you can play with interceptors or scopes, but once again, not sure it would be a good idea
